encrypted-media-respec.html

```diff
@@ -1732,7 +1732,7 @@
                         If the loaded session contains information about any keys (there are <a href="#known-key">known keys</a>), run the <a def-id="update-key-statuses-algorithm"></a> algorithm on the <var>session</var>, providing each key's <a def-id="key-id"></a> along with the appropriate <a>MediaKeyStatus</a>.
                       </p>
                       <p>Should additional processing be necessary to determine with certainty the status of a key, use <a def-id="status-status-pending"></a>.
-                        Once the additional processing for one or more keys has completed, run the <a def-id="update-key-statuses-algorithm"></a> algorithm again with the actual status(es).
+                        Once the additional processing for one or more keys has completed, <a def-id="queue-a-task"></a> to run the <a def-id="update-key-statuses-algorithm"></a> algorithm again with the actual status(es).
```

Comment:
We are already in a task here (line 1725), so we do not need to queue a task and in fact doing so would result in attribute updates happening after promise handlers.